homeassistant/components/signalmessenger/notify.py

```diff
@@ -0,0 +1,72 @@
+"""Signal Messenger for notify component."""
+import logging
+import voluptuous as vol
+from pysignalclirestapi import SignalCliRestApi, SignalCliRestApiError
+
+from homeassistant.components.notify import (
+    ATTR_DATA,
+    PLATFORM_SCHEMA,
+    BaseNotificationService,
+)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = []
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_SENDER_NR = "number"
+CONF_RECP_NR = "recipients"
+CONF_SIGNAL_CLI_REST_API = "url"
+ATTR_FILENAME = "attachment"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_SENDER_NR): cv.string,
+        vol.Required(CONF_SIGNAL_CLI_REST_API): cv.string,
+        vol.Required(CONF_RECP_NR): vol.All(cv.ensure_list, [cv.string]),
+    }
+)
+
+
+def get_service(hass, config, discovery_info=None):
+    """Get the SignalMessenger notification service."""
+
+    sender_nr = config.get(CONF_SENDER_NR)
```

Comment:
We can access the dict directly for required options.
```suggestion
    sender_nr = config[CONF_SENDER_NR]
```